src/BasicLogAnalyser.php

```diff
@@ -300,21 +345,37 @@ private function parseLogEntry($line) {
 				}
 			}
 
+			/**
+			 *
+			 * If there is not CMS detected, we assume WordPress
+			 *
+			 */			
+			if(!isset($result['cms']))
```

Comment:
Code style: Please use braces even when the if() block contains only one statement. For example:
```php
if ($condition) {
     $a = $b;
}
```